docs/content/architectuur/zgw-in-gegevenslandschap.md

```diff
@@ -0,0 +1,188 @@
+In de loop der jaren zijn er diverse doelen voor zaakgericht werken ontstaan: dienstverlening verbeteren, als basis voor het archiveren, het bieden van managementinformatie,.. waarvan het de vraag is of je ze in een modern applicatielandschap nog met zaakgericht werken wilt oplossen. In het geval van een gemeente die met één dominant generiek zaaksysteem werkt ligt de oplossingsrichting voor de hand. In een gegevenslandshap veel minder. Bovendien zien we op vele vlakken een terugtrekkende beweging op het “gemeentebrede” dienstverlenigsconcept, meer richting domeinspecifieke Apps en portalen. Een meer domeingerichte dienstverlening en informatievoorziening (ipv generiek obv zaken) ligt dan meer voor de hand. Opmerking AK  Die twee zijn niet strijdig. Het lijkt er op dat in de redenering  en de voorbeelden het zaakgericht werken als panacee voor alle problemen en behoeften wordt gezien. Evenwel, zaakgericht werken is niet mee of minder dan een manier van procesgericht werken. Een manier die zo ver is uitgewerkt dat anderen dan de uitvoerenden van een proces dat proces ook kunnen begrijpen, op het niveau dat voor een ander volstaat: de hoofdlijnen. Dit ‘esperanto’ blijft een onderscheidend kenmerk van zaakgericht werken en zaken. Je krijgt daarmee twee niveau’s: domeinspecifiek en domeinoverstijgend. Het tweede doen we met zaken.
+
+Enkele voorbeelden:
+1. Een gemeente wil alle Meldingen Openbare Ruimte op een kaart tonen. Vraag je hiertoe alle melding-zaken op met een generiek ZDS-koppelvlak? Wetende dat de AVG dit voor het overgrote deel aan zaken/zaaktypen niet toestaat? Of kies je voor een meldingen-specifiek koppelvlak, dat mogelijk functioneel rijker is: type melding, prioriteit van de melding, tekstuele toelichting…allemaal wellicht nuttig om op een dergelijke kaart te kunnen weergeven. Maar niet of slechts met moeite (zaaktypespecifieke eigenschappen) met een generiek zaken-koppelvlak te realiseren.
+2. Hoe wil je een proceseigenaar/afdelingshoofd van management-informatie voorzien? Op basis van zaken kun je erg generieke metrieken weergeven. Aantal zaken per afdeling, aantal zaken per status, zaken die voor of achter lopen op plandatum etc. Toch is het vergelijken van een subsidiezaak met een vergunnningenzaak appels met peren vergelijken. En zit een manager vergunningen vermoedelijk op heel andere metrieken te wachten: aantal vergunningen per wijk, aantal afhandeluren per bouwvolume-eenheid etc.
+3. Hoe wil je status-informatie weergeven aan de klant? Heel generiek obv enkel voorgedefinieerde “zaakstatussen”, of wil je dit functioneel rijker doen in een domeinspecifiek portal, bijvoorbeeld specifiek voor subsidies?
+
+Het is belangrijk om hier als gemeente even goed bij stil te staan. In veel gevallen zal het nog steeds handig blijken om iets met zaken op te lossen, of misschien beide: generieke informatie via zaken tonen en als aanvulling daarop enkele functioneel rijkere portalen of apps per domein.
+
+### 1.4.8. Notificeren over zaken
```

Comment:
Is er een brengplicht van het component van de wijziging naar het afhandelende systeem?
Of is er een haalplicht van de taakapplicatie?

Of is het een gedeelde verantwoordelijkheid? Het component maakt kenbaar wat de wijzigingen zijn. Het afhandelende systeem checkt regelmatig of er een van de wijzigingen van toepassing is.

Reply:
@joeribekker Dank voor de link naar PubSub! Dat is precies wat ik hierboven bedoel te zeggen :-)

Reply:
Wat gangbaar is is een combinatie: Het component maakt kenbaar dat er wijzigingen zijn en het afhandelende systeem vraagt de nieuwe staat op. Hiermee wordt voorkomen dat informatie die een afhandelend systeem helemaal niet mag weten rondgestuurd wordt, er wordt immers gebruik gemaakt van de bestaande autorisatie om te bepalen welke gegevens wel/niet gestuurd mogen worden.

Reply:
welke link naar PubSub?